Team Members List/Add/Edit/Delete

arghav opened the pull request.

adamstep commented on src/containers/MemberEditContainer.js:
```diff
@@ -0,0 +1,84 @@
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
```
you can use (bindActionCreators)[http://redux.js.org/docs/api/bindActionCreators.html] here to avoid having to explicitly call `dispatch` everywhere.

arghav replied:
Thanks. Switched to bindActionCreators.

adamstep commented on src/containers/TeamMembersContainer.js:
```diff
@@ -0,0 +1,27 @@
+  render() {
+    return (
+      <div className="TeamMembersContainer">
+        {this.props.isEditing && <MemberEditContainer/>}
```
Why not a single ternary operator?
```es6
{this.props.isEditing ? <MemberEditContainer/> : <MemberListContainer/>}
```

arghav replied:
I guess I was sleepy :) Fixed.

adamstep commented on src/components/MemberEdit.js:
```diff
@@ -0,0 +1,56 @@
+            type="text" placeholder="First name"
+            name="firstName"
+            value={this.props.item.firstName}
+            onChange={this.props.handlInputChange} />
```
Typo: `handlInputChange` should be `handleInputChange`

arghav replied:
Thanks. Fixed.

arghav commented on src/components/MemberEdit.js:
```diff
@@ -0,0 +1,56 @@
+          <div className="radio-option">
+            <input type="radio" value={false} name="isAdmin"
+              onChange={() => this.props.handleIsAdminToggle(false)} checked={!this.props.item.isAdmin} />
+              <label>Regular – Can't delete members</label>
```
Fixed.

arghav commented:
@adamstep I have fixed the parts you pointed out. 

Also, added an `ActionBar` component, `PropTypes` check for components, and fixed other minor bugs.

adamstep commented:
Thanks @arghav , looks good!
